Quantum-Control-Applications/Superconducting/quam/Two-Flux-Tunable-Transmons/18_allxy.py

```diff
@@ -199,9 +199,7 @@ def get_prog(qubit, resonator):
             plt.tight_layout()
             plt.pause(0.1)
 
-            data[f"{qb.name}_I"] = I
-            data[f"{qb.name}_Q"] = Q
-            data[f"{qb.name}_figure"] = fig
+            data[qb.name] = {"I": I, "Q": Q, "figure": fig}
```

Comment:
You cannot save a figure in a tree-like structure, it has to be in data = {"figure": fig}, otherwise you get 
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\git\\qua-libs\\Quantum-Control-Applications\\Superconducting\\quam\\Two-Flux-Tunable-Transmons\\DATA\\2024-05-17\\#51_all_xy_125420\\q0\\figure.png'

Reply:
Will try to fix it asap

Reply:
Should be fixed by https://github.com/qua-platform/py-qua-tools/pull/215